solutions/python_azure_sdk/config.json

```diff
@@ -0,0 +1,15 @@
+{
+    // OpenEnclave specific values
+
+    // Whether we are running myst+OE+app in debug mode
+    "Debug": 1,
+    "ProductID": 1,
+    "SecurityVersion": 1,
+
+    // Mystikos specific values
+
+    // The heap size of the user application. Increase this setting if your app experienced OOM.
+    "MemorySize": "3g",
+    "HostApplicationParameters": true,
+    "ForkMode":"pseudo_wait_for_exit_exec"
+}
```

Comment:
Pytest typically use multi-process. Add "NoBrk" configuration, which is required for multi-process app in Mystikos.

Reply:
Fixed.